docs-ui/components/clippedBox.stories.js

```diff
@@ -28,7 +28,7 @@ export const Default = withInfo(
         hoodie 8-bit occupy distillery. Tote bag godard thundercats small batch banjo, DIY
         waistcoat. Glossier poutine VHS put a bird on it listicle deep v letterpress. Tbh
         banjo paleo cred hoodie. Live-edge synth twee, subway tile coloring book woke swag
-        XOXO cornhole glossier neutra hell of lo-fi brooklyn actually. Retro beard
```

Comment:
Lol...

Reply:
coloring book woke swag, you heard it here first folks.